metron-interface/metron-rest-client/src/main/java/org/apache/metron/rest/model/GrokValidation.java

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
I realized after I opened the PR that there were files that had different spacing.  I was planning on waiting until we finished the review process to do this because it will make it easier for you to review the incremental changes.  Is that ok?